kevm-pyk/src/kevm_pyk/kdist/plugin.py

```diff
@@ -27,12 +27,18 @@ def __init__(self, kompile_args: Mapping[str, Any], *, deps: Iterable[str] | Non
     def build(self, output_dir: Path, deps: dict[str, Path], args: dict[str, Any]) -> None:
         verbose = args.get('verbose', False)
         enable_llvm_debug = args.get('enable_llvm_debug', False)
+        debug_build = args.get('debug_build', False)
+        ccopts = args.get('ccopts', [])
 
+        plugin_dir = deps.get('evm-semantics.plugin')
+        if plugin_dir is None:
+            raise ValueError(f'Cannot build target without dependency evm-semantics.plugin already built: {output_dir}')
```

Comment:
This can be ensured by making `evm-semantics.plugin` a dependency of all `KEVMTarget` instances:

```python3
def deps(self) -> tuple[str]:
    return ('evm-semantics.plugin',)
```

and then in `build` simply:

```python3
plugin_dir = deps['evm-semantics.plugin']
```

Reply:
Done.